sdk/storage/storage-blob/src/generated/src/models/index.ts

```diff
@@ -24,7 +23,7 @@ export interface KeyInfo {
 }
 
 /**
- * A user delegation key
+ * A user delegation key.
```

Comment:
Please revert the changes to this file. It is auto-generated code and will get overwritten the next time we generate the same.

We should log a separate issue to track docs for auto-generated entities and fix the swagger as part of that.

Reply:
@XiaoningLiu Is there any other way to improve the docs in the generated models?

Reply:
@joheredi had fixed comments on the generated model by updating the swagger with a transformation in #5742

@HarshaNalluru Is this the suggested way to edit the comments on a generated model?

Reply:
Ah, I should have realized that this was a generated file. I'll revert the changes to this file.

Reply:
Keep a copy of it somewhere, such that if we do decide to update the transformations, we have something to refer to :)

Reply:
Though we can add transformations in readme like #5742, adding too many transformations for the comments doesn't make sense. Ideally, all these comments must be updated in the original swagger JSON file (other languages will also likely get benefitted by this).

Options -
- Add transformations in swagger readme like #5742 and the generated code will have the docs updated.
- Update swagger JSON file ASAP and regenerate the code before GA.
- If we don't want to update the swagger JSON at this point of time(keeping GA in mind), 
We can consider updating the JS Docs like the way Chris did in this PR assuming there won't be any significant updates to the swagger just before GA.
  [ and track the future work to update the swagger JSON with these doc-comments updates and to regenerate the code. ] 
- Not update the docs for code-gen files and keep this work-item for post GA and fix the docs in swagger as a high priority and do a quick minor/patch release.

Reply:
I added a commit to this PR to remove the comments in the generated file, but we can remove that commit if we decide to go ahead with the docs and fast-follow on updating the swagger file.

I would not want to add transformations based on the sheer number of updates we'd have to make.

Reply:
I agree. I'd suggest option-3 (and making a separate PR for it just for the generated files).

cc - @XiaoningLiu